{"name": "test_sample2", "status": "failed", "statusDetails": {"message": "AssertionError: assert 5 == 6", "trace": "def test_sample2():\n>       assert 5 == 6\nE       assert 5 == 6\n\nPythonAPI\\test_API005.py:14: AssertionError"}, "start": 1702461446590, "stop": 1702461446590, "uuid": "2971575a-e6e9-421a-9d25-1dc79d6f59fe", "historyId": "cea2f82a2a7b7374d15ec0a2b4eeec33", "testCaseId": "cea2f82a2a7b7374d15ec0a2b4eeec33", "fullName": "PythonAPI.test_API005#test_sample2", "labels": [{"name": "parentSuite", "value": "PythonAPI"}, {"name": "suite", "value": "test_API005"}, {"name": "host", "value": "DESKTOP-HP3DQ6Q"}, {"name": "thread", "value": "4612-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "PythonAPI.test_API005"}]}